### ISSUE TITLE
Error on executing Abricotine latest version

### ISSUE THREAD

**Issue by insanetesterftw** [brrd/Abricotine#53]
Using Windows 7 64 bit
Using (or trying to use) Abricotine, latest version.

On first run, it said it would update configuration files to the latest version.

On second run gives the following error: 
![2016-03-29_04-43-38](https://cloud.githubusercontent.com/assets/11148841/14097974/35b974a0-f56f-11e5-99b1-ffed426e04e0.jpg)

**Comment by brrd** [comment]
Please open a terminal and try running: `C:\path\to\Abricotine.exe --reset`
Is this fixing the issue ?

**Comment by dieris** [comment]
I have the same issue on Windows 8 64 bit. Running `C:\path\to\Abricotine.exe --reset` reset causes the same error

**Comment by fvsch** [comment]
Same issue here, on Windows 10. Starting with `--reset` didn't change the result.

**Comment by brrd** [comment]
Were you using previous version (0.2.2) before you tried Abricotine 0.3.0?

**Comment by fvsch** [comment]
No, first download and first try (well, tries) to launch it. First launch there was a pop-up about the config being out-of-sync, asked me if I wanted to update it and I picked yes. Then this error pop-up, and same error on all subsequent launches.

**Comment by brrd** [comment]
Hmm... :confused: I'm going to try to find a PC with Windows to reproduce and fix this bug.
You can use the previous version until 0.3.1 is released.
Thanks for you report.

**Comment by DigitalChiller** [comment]
same thing here, but on linux 64bit, cinnamon desktop, version 0.3.0, first time with abricotine.
when running it from the command line gives this:
>(electron) ipc module is deprecated. Use ipcMain module instead.

after a short while though, the error popup closes, the program opens and the terminal spits this out:
>(electron) loadUrl is deprecated. Use loadURL instead.

now the error doesn't pop up anymore and the programm opens faster. I like it so far :)

**Comment by francesca-c** [comment]
I also had the same issue, running Linux Mint 17.3 Cinnamon 64 bit. I tried running ./Abricotine --reset, and it did the same, both the Javascript error "cannot read property "run" of null" and 
"(electron) ipc module is deprecated. Use ipcMain module instead."
I tried a dozen times without success. 

Then it gave the error "(electron) loadUrl is deprecated. Use loadURL instead." 
And it suddenly started working. I didn't do anything different. Mystery...

This is using version v0.35.1.

The program looks really nice though. Thanks!
